crates/solidity/testing/sourcify/src/command.rs

```diff
@@ -51,15 +48,24 @@ pub struct ShowCombinedResultsCommand {
     pub results_file: PathBuf,
 }
 
+#[allow(clippy::struct_excessive_bools)]
 #[derive(Debug, Parser)]
 pub struct TestOptions {
     /// Run bindings tests.
     #[arg(long, default_value_t = false)]
     pub check_bindings: bool,
 
+    /// Run new binder tests.
+    #[arg(long, default_value_t = false)]
+    pub check_new_binder: bool,
```

Comment:
nit: I suggest renaming both options to `check_binder_v1` and `check_binder_v2` for disambiguating.